### ISSUE TITLE
use las file

### ISSUE THREAD

**Issue by Sassimion** [TianzhongSong/PointNet-Keras#3]
Hi I'm a newbie. is it possible to use las data with this code?

**Comment by ghost** [comment]
> Hi I'm a newbie. is it possible to use las data with this code?

Yes, same question with me... how to use this code with las data , since I am a newbie dont have much exploration. Any help will me much appreciated.
